README.rst

```diff
@@ -322,6 +322,20 @@ Python iRODS Client Settings File
 
 As of v1.1.9, Python iRODS client configuration can be saved in, and loaded from, a settings file.
 
+If the settings file exists, each of its lines contains (a) a dotted name identifying a particular configuration setting
+to be assigned within the PRC, potentially changing its runtime behavior; and (b) the specific value, in Python "repr"-style
+format, that should be assigned into it.
+
+An example follows:
+
+  data_objects.auto_close   True
+
+New dotted names may be created following the example of the one valid example created thus far,
+code:`data_objects.auto_close`, initialized in :code:`irods/client_configuration/__init__.py`. Each such name should correspond
+to a globally set value which the PRC routinely checks when performing the affected library function.
+
+The rule is: if you're adding a configurable behavior to PRC, teach the client_configuration module to load and save it.
```

Comment:
this line is not for readers/users, it's for developers.   please remove it.

Reply:
The line that says "The rule is: ..." ?

Reply:
yes

Reply:
Btw, where is our dev documentation for PRC ... ; ) ?

Reply:
not today :P

Reply:
done